manifests/kustomize/base/metadata/metadata-writer-deployment.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: metadata-writer
+  labels:
+    app: metadata-writer
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: metadata-writer
+  template:
+    metadata:
+      labels:
+        app: metadata-writer
+    spec:
+      containers:
+      - name: main
+        image: gcr.io/ml-pipeline/metadata-writer:0.1.40
```

Comment:
please refer manifests/kustomize/base/kustomization.yaml to let it be central controlled. Easy for each on-call's weekly release.

Reply:
I opened kubeflow/pipelines#2882 addressing that.